src/HaskellCI/Cli.hs

```diff
@@ -23,6 +25,7 @@ data Command
     = CommandTravis FilePath
     | CommandBash FilePath
     | CommandGitHub FilePath
+    | CommandSourcehut (SourcehutOptions (Maybe String))
```

Comment:
Don't introduce separate SourceHut options. Everything is configured through `Config`.

Reply:
Yeah I was about to ask about this since it only seems to introduce problems (especially parseOptions). Will do.

Reply:
Done